conf/telemetry/default_rotorcraft.xml

```diff
@@ -13,7 +13,7 @@
       <message name="ALIVE"                    period="2.1"/>
       <message name="INS_REF"                  period="5.1"/>
       <message name="ROTORCRAFT_NAV_STATUS"    period="1.6"/>
-      <message name="WP_MOVED"                 period="1.3"/>
```

Comment:
Why is this needed ? WP_MOVED is normally sent as an acknowledgment when a WP is moved. The message in telemetry file should be a low freq update in case a message is missed or when the ref point is changed.
This might significantly increase the telemetry rate, so I don't think it is good to have it in the default file. If it is really needed, then make a dedicated file, or make a module that directly send the WP_MOVED acknowledgment message at the correct rate.

Reply:
The avoidance module used the waypoints_set_ functions which do not send this acknowledgement. I was not aware of this so I increased the update rate.

I have implemented the missing waypoint_move_ functions which *do* send this message, and restored the WP_MOVED update rate to its original value.